content/04.experiment-types.md

```diff
@@ -16,4 +16,146 @@ Proteomics experiments can be both qualitative and quantitative.
+If done correctly, HDX can produce precise protein structure measurements with high reproducibility.
+Masson **et al.** gave recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique [@doi:10.1038/s41592-019-0459-y].
+
+3. Fast photochemical oxidation of proteins (FPOP) [@doi:10.1074/jbc.REV119.006218].
+FPOP is a laser based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins.
+A laser produces 248 nm light to cause hydrogen peroxide to become hydroxyl radicals [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018]. 
+The flow rate of solution through the capillary and laser frequency are adjusted so that each protein molecule is radiated only once. 
+After they are radiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling. 
+Control samples are made through running the sample through the flow system without any radiation. 
+Another experimental control is done by adding a radical scavenger with the purpose of preventing over oxidation [@doi:10.1002/pro.3408; @doi:10.1021/ac901054w].
+FPOP has wide application for proteins, to include being able to report fast protein folding and protein transient dynamics.
+
+4. Protein Painting [@doi:10.1038/ncomms5413; @doi:10.1074/jbc.RA118.007310].
+Protein painting uses "molecular paints" to noncovalently coat the solvent accessible surface of proteins. 
+These paint molecules will coat the protein surfaces but will not have access to the hydrophobic cores or protein-protein interface regions that solvents cannot access. 
+If the "paint" cover free amines of lysine side chains, the parts that are painted will not be subjected to trypsin cleavage, while the unpainted areas will. 
+After proteolysis, the peptides samples will be subjected to MS. 
+The samples will only show the unmodified opposing regions in the protein-protein interaction at the time of the painting. 
+
+5. LiP-MS (limited proteolysis mass spectrometry) [@doi:10.1016/j.tibs.2020.05.006; @DOI:10.1038/nprot.2017.100; @doi:10.1038/s41596-022-00771-x; @doi:10.1038/nprot.2017.100].
+Limited proteolysis coupled to mass spectrometry (LiP-MS) is a method that tracks structural changes in complex proteoms in response to stimuli.
+This method relies on the usage of stimuli (small molecules and proteinase K (PK)) as the additions would cause tertiary and quaternary structures. 
+Samples that undergo LiP-MS will show at where the structural changes will happen and give an understanding as to why PK bind to that spot.
```

Comment:
```suggestion
LiP-MS can therefore provide a somewhat unbiased view of structural changes at the proteome scale.
This may be useful when attempting to discover a mechanism for some treatment that causes a phenotype. 
```